### PR TITLE
[FIX] Iso format for v3 is called twice

### PR DESCRIPTION
**Pull request by flachica**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,6 @@
 sudo: false
 language: python
 python:
-  - "2.7"
   - "3.6"
```
Comment (lemoustachiste) [inline .travis.yml:4]: why removing the 2.7 version?

File: cert_tools/helpers.py [hunk 2]
```diff
@@ -57,5 +57,4 @@ def encode(num, alphabet=BASE62):
 
 
 def create_iso8601_tz():
-    ret = datetime.now(timezone.utc).isoformat()[:-13]+'Z'
-    return ret.isoformat()
\ No newline at end of file
+    return datetime.now(timezone.utc).isoformat()[:-13]+'Z'
```
Comment (lemoustachiste) [inline cert_tools/helpers.py:60]: that looks good, now that I look at the other code, it was indeed a strange setup.

**Review by lemoustachiste — COMMENTED** [review]
@lparkerlm can you weigh in on the removal of python 2.7?

File: .travis.yml (hunk 1 above)
Comment (flachica) [reply .travis.yml:4]: To be able to pass the tests

**Review by lemoustachiste — CHANGES_REQUESTED** [review]
@flachica we had a discussion with @lparkerlm and as it stands, removing python 2.7 cannot be accepted as part of this PR. We need more time to flesh out the issue with this version, and decide if we maintain or remove support. 

In my opinion your code looks ok, and if it works with Python 3, then I think it's good enough for the current purpose. Python 2.7 issue has been lingering for a while, so we would need to take a proper look at it.

**Comment by antonellopasella** [comment]
Updated without Python3 removal in https://github.com/blockchain-certificates/cert-tools/pull/51

**Pull request closed without merging**